Client/mods/deathmatch/logic/luadefs/CLuaWorldDefs.h

```diff
@@ -130,4 +130,6 @@ class CLuaWorldDefs : public CLuaDefs
     static std::variant<bool, float, CLuaMultiReturn<float, float, float>> GetWorldProperty(eWorldProperty property);
     static bool SetWorldProperty(eWorldProperty property, float arg1, std::optional<float> arg2, std::optional<float> arg3);
     static bool ResetWorldProperty(eWorldProperty property);
+    static bool SetVolumetricShadowsEnabled(bool enable) noexcept;
```

Comment:
Do you have any `void` type lua functions?

Reply:
I will test later with void because last time I used argument parser with void type the build was failing and gives error function has no return value

Reply:
> Do you have any `void` type lua functions? How would that work here?

I forgot that @Proxy-99 uses ArgumentParser here, sorry.

Reply:
> I will test later with void because last time I used argument parser with void type the build was failing and gives error function has no return value

I forgot that you are using ArgumentParser here, the function need to have a return value, sorry.